hazelcast/src/test/java/com/hazelcast/query/impl/predicates/PredicatesTest.java

```diff
@@ -252,6 +253,12 @@ public void testILike() {
     }
 
     @Test
+    public void testILike_Id() {
+        ILikePredicate predicate = (ILikePredicate) ilike(ATTRIBUTE, "Java_World");
+        assertEquals(PredicateDataSerializerHook.ILIKE_PREDICATE, predicate.getId());
```

Comment:
wouldn't it make more sense for test to use the actual ID instead of linking the constant? so if you change the constant by accident then the test fails.